app/assets/stylesheets/common/base/topic-admin-menu.scss

```diff
@@ -39,6 +39,18 @@
   @include breakpoint(mobile-extra-large) {
     width: 100%;
     padding: 0;
+    padding-bottom: 10px;
```

Comment:
instead I recommend using this pattern https://css-tricks.com/the-notch-and-css/ unless you want the padding unconditionally.